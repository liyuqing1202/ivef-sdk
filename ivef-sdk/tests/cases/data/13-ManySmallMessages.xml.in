<MSG_IVEF> <Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/> <Body> <Logout> </Logout> </Body> </MSG_IVEF> <MSG_IVEF> <Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/> <Body> <Logout> </Logout> </Body> </MSG_IVEF> <MSG_IVEF> <Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/> <Body> <Logout> </Logout> </Body> </MSG_IVEF> <MSG_IVEF> <Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/> <Body> <Logout> </Logout> </Body> </MSG_IVEF> <MSG_IVEF> <Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/> <Body> <Logout> </Logout> </Body> </MSG_IVEF> <MSG_IVEF> <Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/> <Body> <Logout> </Logout> </Body> </MSG_IVEF> <MSG_IVEF> <Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/> <Body> <Logout> </Logout> </Body> </MSG_IVEF> <MSG_IVEF> <Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/> <Body> <Logout> </Logout> </Body> </MSG_IVEF> <MSG_IVEF> <Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/> <Body> <Logout> </Logout> </Body> </MSG_IVEF>
