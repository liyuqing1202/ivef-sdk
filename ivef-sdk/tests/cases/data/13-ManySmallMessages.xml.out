TestApp ready for input
MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 0.2.5
    Body
        Logout
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/>
<Body>
<Logout/>
</Body>
</MSG_IVEF>

MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 0.2.5
    Body
        Logout
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/>
<Body>
<Logout/>
</Body>
</MSG_IVEF>

MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 0.2.5
    Body
        Logout
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/>
<Body>
<Logout/>
</Body>
</MSG_IVEF>

MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 0.2.5
    Body
        Logout
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/>
<Body>
<Logout/>
</Body>
</MSG_IVEF>

MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 0.2.5
    Body
        Logout
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/>
<Body>
<Logout/>
</Body>
</MSG_IVEF>

MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 0.2.5
    Body
        Logout
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/>
<Body>
<Logout/>
</Body>
</MSG_IVEF>

MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 0.2.5
    Body
        Logout
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/>
<Body>
<Logout/>
</Body>
</MSG_IVEF>

MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 0.2.5
    Body
        Logout
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/>
<Body>
<Logout/>
</Body>
</MSG_IVEF>

MSG_IVEF
    Header
        MsgRefId = 6289936C-DFAC-11E0-B7BA-B2E24824019B
        Version = 0.2.5
    Body
        Logout
<MSG_IVEF>
<Header MsgRefId="6289936C-DFAC-11E0-B7BA-B2E24824019B" Version="0.2.5"/>
<Body>
<Logout/>
</Body>
</MSG_IVEF>

TestApp shutting down
